src/test/miner_tests.cpp

```diff
@@ -23,6 +23,7 @@
 #include <util/system.h>
 #include <util/time.h>
 #include <validation.h>
+#include <versionbits.h>
```

Comment:
22630: too early for this, we don't have 22277 which introduced the issue

Reply:
dropped 22630